user-guide/Reference/faq/IDs_of_views_elements_services_redundancy_groups_and_parameters.md

```diff
@@ -38,7 +38,7 @@ To find this ID in DataMiner Cube:
 
 On the element card containing the parameter, double-click the parameter, or right-click the parameter and select *Open*.
```

Comment:
This should be changed to a complete procedure, because the *Details* tab is not shown by default if trending is active on the parameter:
1. Open the element card containing the parameter.
1. Double-click the parameter, or right-click the parameter and select *Open*.
1. On the *Details* tab, look for the ID displayed below the parameter name.